readthedocs/core/utils/__init__.py

```diff
@@ -222,6 +222,7 @@ def safe_makedirs(directory_name):
     try:
         os.makedirs(directory_name)
     except OSError as e:
-        if e.errno == errno.EEXIST:
+        if e.errno == errno.EEXIST:  # 17, FileExistsError
```

Comment:
Can we rewrite this as `raise if not e.errno == errno.EEXIST`?

Reply:
I think we can't. Doesn't look like valid syntax to me.

```
>>> raise if None is []
  File "<stdin>", line 1
    raise if None is []
           ^
SyntaxError: invalid syntax
>>> 
```

```
>>> raise if None is [] else 2
  File "<stdin>", line 1
    raise if None is [] else 2
           ^
SyntaxError: invalid syntax
```

Reply:
OK. We should at least be able to do:

```
if not e.errno == errno.EEXIST:
   raise
```

Reply:
oh, maybe we can write it like

```
if e.errno != errno.EEXIST:
  raise
```

is that what you ment?

Reply:
Ha, even better :D